src/commands/lightning/dev/component.ts

```diff
@@ -5,48 +5,80 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import fs from 'node:fs';
+import path from 'node:path';
 import { SfCommand, Flags } from '@salesforce/sf-plugins-core';
 import { Messages } from '@salesforce/core';
 import { cmpDev } from '@lwrjs/api';
+import { PromptUtils } from '../../../shared/promptUtils.js';
 
 Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
 const messages = Messages.loadMessages('@salesforce/plugin-lightning-dev', 'lightning.dev.component');
 
-export type LightningDevComponentResult = {
-  path: string;
-};
+const MODULES_DIR = './force-app/main/default/lwc';
```

Comment:
probably not the best way to do this - we'll want this to work on windows (not sure what machines greg is demoing on)